### PR TITLE
AP-1589: Geckoboard provider update

### PR DESCRIPTION
**Pull request by colinbruce**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-1589)

Add a new geckoboard report to improve the handling and tracking of provider creation

## Checklist

Before you ask people to review this PR:

- [ ] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/models/dashboard/single_object/provider_data.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          {
+            reference: @provider.id,
+            timestamp: @provider.created_at,
+            firm: @provider.firm.name,
+            count: @provider.legal_aid_applications.count
+          }
+        ]
+      end
+
+      # private
```
Comment (mpw5) [inline app/models/dashboard/single_object/provider_data.rb:28]: Should this be commented out?

File: spec/jobs/dashboard/provider_data_job_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      let(:dataset) { double Geckoboard::Dataset }
+
+      before do
+        allow(Geckoboard).to receive(:client).and_return(geckoboard_client)
+        allow(geckoboard_client).to receive(:ping).and_return(true)
+        allow(geckoboard_client).to receive(:datasets).and_return(datasets_client)
+      end
+
+      describe '#perform' do
+        it 'calls the runs the geckoboard feedback updater' do
```
Comment (mpw5) [inline spec/jobs/dashboard/provider_data_job_spec.rb:20]: Really nitpicky - should this be `it 'calls the geckoboard feedback updater' do`?

**Review by mpw5 — APPROVED** [review]
LGTM - couple of nitpicky comments, sorry, but no blockers

**Comment by colinbruce** [comment]
Thanks @mpw5 turns out I forgot to push my last commit anyway, so I've fixed both of these issues and will push shortly :)

**Review by mpw5 — APPROVED** [review]
LGTM!

**Pull request merged**